battleship-logic/src/main/java/dreamteam/battleship/logic/arbiter/Arbiter.java

```diff
@@ -27,4 +27,10 @@
      * won the game, because he destroyed all opponent's ships.
      */
     public abstract boolean isWinner();
+
+    /**
+     *
+     * @return number of ships that are alived
+     */
```

Comment:
Maybe not necessary to write a comment when function name is clear and say what it does ;)